rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -429,7 +429,7 @@ namespace rsx
 				{
 					// Ignore addresses outside the usable [0, 467] range
 					rsx_log.warning("Invalid transform register index (load=%u, index=%u, count=%u)", load, index, count);
-					rcount -= max - (468 * 4);
+					rcount = (max - count >= 468 * 4) ? 0 : rcount - max - (468 * 4);
```

Comment:
Actually,  I don't like this statement. It is barely readable. Is this intentional?

Reply:
I can move constexpr 468 * 4, other than that not much can be changed here.

Reply:
Done.

Reply:
Thanks. The issue for me was the lack of concise bracing around every term, but multiline also works.